client/my-sites/customer-home/cards/features/go-mobile/style.scss

```
@@ -102,13 +109,17 @@
 		color: var(--studio-jetpack-green-40);
 		margin-top: 0;
 		text-decoration: underline;
+		text-underline-offset: 3px;
+		font-size: rem(15px);
```

Does this need a note? 
```suggestion
		font-size: rem(15px); //typography-exception
```

---

Ah, yes, that rule had slipped my mind! I've fixed that in https://github.com/Automattic/wp-calypso/pull/69000/commits/4b2c36625acce914e307a0a175fbcf4f4ae62866 and adding the reference for that rule here for posterity:

https://github.com/Automattic/wp-calypso/blob/1fbc67710c8164f0380761e8d3a5fb0d75a5bec8/packages/typography/styles/variables.scss#L31-L34